plugins/sprites.py

```diff
@@ -1,75 +1,64 @@
 from __future__ import annotations
 
 import random
-import re
 from typing import TYPE_CHECKING
 
 from imageprobe.errors import UnsupportedFormat
 from sqlalchemy.exc import SQLAlchemyError
 from sqlalchemy.sql.expression import func
 
 import databases.veekun as v
-import utils
 from database import Database
 from plugins import command_wrapper
+from typedefs import JsonDict
+from utils import get_ps_dex_entry, image_url_to_html, to_id
```

Comment:
I'm aware that we usually use `import utils` but writing `utils.to_id` multiple times in the same line was a bit verbose.